tests/tests.cc

```diff
@@ -29,14 +29,14 @@
 
 int main()
 {
-    // tests::connection_test st;
-    // st.run();
+    tests::connection_test st;
+    st.run();
 
     // tests::conversion_test ct;
     // ct.run();
 
-    tests::mission_test mt;
-    mt.run();
+    // tests::mission_test mt;
+    // mt.run();
```

Comment:
I agree that this is pretty dirty. I'll make changes to have them executed one after the other instead of using parameters. It seems more straightforward. What do you think?